chore: update information on how to contribute and what is involved in maintainership (#214)
sanity-io/locales: CONTRIBUTING.md

# Contributing

Contributions are always welcome and there are multiple ways to join in from being a maintainer of a particular locale to submitting one-off improvement suggestions, big or small. Maintainers are tagged to help review automated pull-requests from AI and receive a token of appreciation from Sanity.

## Getting started

Before contributing, please read our [code of conduct](https://github.com/sanity-io/locales/blob/main/CODE_OF_CONDUCT.md).

## Suggesting improvements

> You can easily add suggestions to existing pull requests from AI and other contributers without checking out code, here is a quick demo:

Reviewer: ```suggestion
You can easily add suggestions to existing pull requests from AI and other contributors without checking out code, here is a quick demo:
```

[![Demo of making suggestions](https://github.com/sanity-io/locales/assets/38528/6b49bd12-ac44-4471-881f-b3168e97b5f7)](https://www.loom.com/share/4ae37eb58985467fb9f68498ce669038?sid=a5d3dc37-22d2-4a78-8452-5ae7039f11e1)

### Submitting your own PRs

Make sure you have Node.js version 18.3 or newer, as well as pnpm.

```bash
git clone git@github.com:sanity-io/locales.git
cd locales
pnpm install
pnpm run build
```

> The strings are located in the src folder of each plugin, where the file name represents the "namespace". Send a pull request if you have suggestions for improvements, and feel free to add yourself to the contributors array in locales/registry.ts. If you want to help review new translations, add yourself to the maintainers array instead, see below for more information. After doing your changes, run pnpm run reconcile and add the changes to version control.

Reviewer: ```suggestion
The strings are located in the `src` folder of each plugin, where the file name represents the "namespace". Send a pull request if you have suggestions for improvements, and feel free to add yourself to the contributors array in locales/registry.ts. If you want to help review new translations, add yourself to the maintainers array instead, see below for more information. After doing your changes, run pnpm run reconcile and add the changes to version control.
```

## Adding a new locale

Hooray! We appreciate your contribution and love seeing new locales added! Initially, you probably do not want to hand-translate every single string in the studio. Instead, simply [request a new locale](https://github.com/sanity-io/locales/issues/new?assignees=&labels=&template=new-locale-request.md&title=Locale+request%3A+) and we will run an initial AI translation for you. Once this is done, you can contribute edits/improvments to it and optinally be its maintainer.

> Here is a full walktrhough of how we use AI to help us create language plugins and keep them updated, and how maintainers and contributers help

Reviewer: ```suggestion
Here is a full walkthrough of how we use AI to help us create language plugins and keep them updated, and how maintainers and contributors help
```

   - Note: The reconciliation script will make changes to `pnpm-lock.yaml`, `apps/studio` and possibly `locales/registry.ts` (if not sorted correctly). This is expected, and should be commited.
7. Push your changes to a fork and send a pull request!

## Maintaining a locale

> Want to help maintain a plugin? Awesome! We appreciate your help! To get started can either [request to be added as a maintainer](https://github.com/sanity-io/locales/issues/new?assignees=&labels=&projects=&template=request-maintainer.md&title=Maintainer%3A+LOCALE-ID), or add yourself as one by following these steps:

Reviewer: ```suggestion
Want to help maintain a plugin? Awesome! We appreciate your help! To get started, either [request to be added as a maintainer](https://github.com/sanity-io/locales/issues/new?assignees=&labels=&projects=&template=request-maintainer.md&title=Maintainer%3A+LOCALE-ID), or add yourself as one by following these steps:
```
